tests/Unit/CssInlinerTest.php

```diff
@@ -2618,6 +2618,59 @@ public function inlineCssNotInDebugModeKeepsInvalidOrUnrecognizedSelectorsInMedi
         self::assertContainsCss($css, $subject->render());
     }
 
+    /**
+     * @test
+     */
+    public function getMatchingUninlinableSelectorsReturnsMatchingUninlinableSelector()
+    {
+        $subject = $this->buildDebugSubject('<html><p>foo</p></html>');
+        $subject->inlineCss('p:hover { color: green; }');
+
+        $result = $subject->getMatchingUninlinableSelectors();
+
+        static::assertContains('p:hover', $result);
```

Comment:
I propose we put the `p:hover` into a variable to make the connection more explicit. Ditto in the other tests.

Reply:
I've refactored the tests to use a data provider, which achieves this, and also allows additional tests to be added more easily if required - in fact I've added one more test for completeness.